lib/shared-method.js

```diff
@@ -701,3 +716,15 @@ SharedMethod.prototype.addAlias = function(alias) {
     this.aliases.push(alias);
   }
 };
+
+// Include support for following Number methods to support Node v.0.10.x
+Number.isInteger = Number.isInteger || function(value) {
+  return typeof value === 'number' &&
+    isFinite(value) &&
+    Math.floor(value) === value;
+};
+
+Number.isSafeInteger = Number.isSafeInteger || function(value) {
+  Number.MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER || Math.pow(2, 53) - 1;
```

Comment:
Please move this line outside of `Number.isSafeInteger` function, no need to run it every time `isSafeInteger` is called.